src/assets/css/forgotPage.css

```diff
@@ -0,0 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
```

Comment:
forgotPage.css => ForgotPage.css

move this css files to the resp folder refer @Parasuramandckap